docs/News.md

```diff
@@ -1,3 +1,49 @@
+    * [Chromophobe Renal Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=chrcc_tcga_gdc) *66 samples*
+    * [Colon Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=coad_tcga_gdc) *463 samples*
+    * [Cutaneous Melanoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=skcm_tcga_gdc) *473 samples*
+    * [Diffuse Glioma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=difg_tcga_gdc) *530 samples*
+    * [Diffuse Large B-Cell Lymphoma, NOS (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=dlbclnos_tcga_gdc) *48 samples*
+    * [Endometrial Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ucec_tcga_gdc) *548 samples*
+    * [Esophageal Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=esca_tcga_gdc) *186 samples*
+    * [Glioblastoma Multiforme (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=gbm_tcga_gdc) *610 samples*
+    * [Head and Neck Squamous Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hnsc_tcga_gdc) *530 samples*
+    * [Hepatocellular Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hcc_tcga_gdc) *379 samples*
+    * [High-Grade Serous Ovarian Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hgsoc_tcga_gdc) *602 samples*
+    * [Invasive Breast Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=brca_tcga_gdc) *1103 samples*
+    * [Lung Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=luad_tcga_gdc) *571 samples*
+    * [Lung Squamous Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=lusc_tcga_gdc) *503 samples*
+    * [Miscellaneous Neuroepithelial Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=mnet_tcga_gdc) *184 samples*
+    * [Non-Seminomatous Germ Cell Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=nsgct_tcga_gdc) *156 samples*
+    * [Pancreatic Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=paad_tcga_gdc) *186 samples*
+    * [Papillary Renal Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prcc_tcga_gdc) *292 samples*
+    * [Papillary Thyroid Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thpa_tcga_gdc) *515 samples*
+    * [Pleural Mesothelioma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=plmeso_tcga_gdc) *87 samples*
+    * [Prostate Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prad_tcga_gdc) *501 samples*
+    * [Rectal Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=read_tcga_gdc) *171 samples*
+    * [Renal Clear Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ccrcc_tcga_gdc) *537 samples*
+    * [Soft Tissue (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=soft_tissue_tcga_gdc) *264 samples*
```

Comment:
@jamesqo, should this also be `Soft Tissue Cancer`?

Reply:
@rmadupuri Ah yes, good catch!